tests/fixtures/insights/noApplicants/sample_applicants.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
I don't think it's necessary to create this! In the fixture itself, you can always check for an empty array instead of loading a file and reading it!